doc/filesystem.rst

```diff
@@ -192,8 +192,55 @@ in the specific subdirectory.  This mimics the POSIX behavior for
 directory traversal most C programmers are used to.
 
 
-Uploading files to file system
-------------------------------
+Uploading files to file system (new)
```

Comment:
OK; good point (Thanks)

Reply:
Hm, I don't understand what happend here. You changed this, but inside the "resolved" view I still see my old version ???
I prefer You Version (with "combined firmware and filesystem" instead of "new"), but that's not shown here ???

Reply:
OK, because that seams to be true for all such comments here, I integrate theses changes into my local copy and commit theses changes later..
If this isn't the right way, please let me know.